src/firmware/host/types/snp.rs

```diff
@@ -122,6 +126,19 @@ impl CertTableEntry {
     pub fn new(cert_type: CertType, data: Vec<u8>) -> Self {
         Self { cert_type, data }
     }
+
+    /// Builds a Kernel formatted CertTable for sending the certificate content to the PSP.
+    pub fn cert_table_to_vec_bytes(table: &[Self]) -> Result<Vec<u8>, CertError> {
+        FFI::types::CertTableEntry::uapi_to_vec_bytes(&table.to_vec())
```

Comment:
Didn't we plan to update `uapi_to_vec_bytes` to receive a reference instead of a vector? If so, we will need to adjust this to not be sending a vector.
```suggestion
        FFI::types::CertTableEntry::uapi_to_vec_bytes(table)
```